manifests/root-reconciler-base-cluster-role.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: configsync.gke.io:root-reconciler
+  labels:
+    configmanagement.gke.io/system: "true"
+    configmanagement.gke.io/arch: "csmr"
+rules:
+- apiGroups: ["configsync.gke.io"]
+  resources: ["rootsyncs"]
+  verbs: ["get","list","watch","update","patch"]
+- apiGroups: ["configsync.gke.io"]
+  resources: ["rootsyncs/status"]
+  verbs: ["get","list","watch","update","patch"]
```

Comment:
It needs to be able to access RepoSyncs (RSyncs of Rsyncs), Secrets, and ConfigMaps as well?

Reply:
It's not clear to me that we should grant those permissions as a baseline. It's possible that a platform admin doesn't want to grant permission for a given RootSync to manage RepoSyncs